package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -230,7 +277,7 @@ def parse(self, **kwargs):
 
         return top
 
-    def read_DATA_timestep(self, n_atoms, TS_class, TS_kwargs):
+    def read_DATA_timestep(self, n_atoms, TS_class, TS_kwargs, atom_style):
```

Comment:
If this is a user-accessible function and the signature changed then putting it into 0.17.1 is a no-no because you're potentially breaking user code. Can you make it kwarg and default to `None`?

Also, a `.. versionchanged: 0.17.1` should be added.